pallets/parachain-staking/src/lib.rs

```diff
@@ -843,26 +864,44 @@ pub mod pallet {
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
 		fn on_initialize(n: T::BlockNumber) -> Weight {
-			let mut round = <Round<T>>::get();
-			if round.should_update(n) {
-				// mutate round
-				round.update(n);
+			let mut current_round = <Round<T>>::get();
+			if current_round.should_run_election(n) {
+				// execute all delayed nominator exits
+				Self::execute_nominator_exits(current_round.current + 1u32);
```

Comment:
```suggestion
				Self::execute_nominator_exits(current_round.next());
```
consider adding a function that returns `current_round.current.saturating_add(1u32)` because this is used in 3 places

Reply:
done